### ISSUE TITLE
forbidden_void_types: checks of structures

### ISSUE THREAD

**Issue by mbtools** [abaplint/abaplint#1294]
![image](https://user-images.githubusercontent.com/59966492/92555429-a1cd5b80-f235-11ea-8836-d47167b316bc.png)

In this case, structure BAPIDEFAUL was used in the program:
`DATA ls_data TYPE BAPIDEFAUL.`

```
    "forbidden_void_type": {
      "check": ["^char.*$", "^sydatum$", "^syindex$", "^sytabix$", "^syuzeit$"]
    },

```

1. Dependency seems to resolve all fields of this SAP structure into the domains and then error on used types (CHAR8 above). Should we really check SAP structures like this?
1. Error marks all fields as "void" although only one field is wrong.

**Comment by larshp** [comment]
hmm

whats the dependency depth set to?

https://github.com/abaplint/abaplint-sci-client

**Comment by larshp** [comment]
@mbtools try setting this flag, https://github.com/abaplint/abaplint-sci-client/pull/149

**Comment by mbtools** [comment]
solved in latest version (no need for flag)
